source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+So with :file:`nano /etc/resolv.conf` we create the new file and enter:
+
+.. code-block:: bash
+
+	nameserver 127.0.0.1
+	options edns0
+
+We then need to stop and disable the currently running pre-installed resolver. Note that you cannot go to new websites until the next step after this, as you have no DNS resolver assigned for the system.
+
+.. code-block:: bash
+
+	sudo systemctl disable systemd-resolved.service
+	sudo systemctl stop systemd-resolved
+
+Now the operating system should use our Unbound instance as default. A quick test a :command:`dig` without specifying the address of the Unbound server should give the same result as specifying it did above (with ``@127.0.0.1``).
+
+.. code-block:: bash
+
+	dig example.com
+
+Note that the "SERVER" section in the output from :command:`dig` should also contain the local IP address of our server.
+
+.. code-block:: bash
+
+	;; SERVER: 127.0.0.1#53(127.0.0.1)
+
+Setting up for the rest of the network
+--------------------------------------
+
+While we currently have a working instance of Unbound, we need it to be reachable from within our entire network. With that comes the headache of dealing with (local) IP addresses. It’s likely that your home router distributed local IP addresses to your devices. If this is the case (i.e. you didn’t change it by hand), they should be :rfc:`1918` ranges:
+
+.. code-block:: bash
+
+	10.0.0.0 - 10.255.255.255 (10/8)
+	172.16.0.0 - 172.31.255.255 (172.16/12)
+	192.168.0.0 - 192.168.255.255 (192.168/16)
+
+To find the IP address of the machine that is running Unbound, we use:
+
+.. code-block:: bash
+
+	hostname --all-ip-addresses
+
+If you just have one IP address as output from the :command:`hostname` command that will be the correct one. If you have multiple IP addresses the easiest way which IP address to use, is to find out which connection goes to your home router. Keep in mind that finding the wrong IP address here this can be a source of connectivity errors further on. For purpose of this tutorial we imagine that our home router has ``10.0.0.1`` as IP address, and our resolver machine (the machine that is running our Unbound instance) has ``10.0.0.2``, which we will get into in the next section.
+
+As prerequisite for the next step we need to configure our Unbound instance to be reachable from other devices than only the machine on which the instance is running. The full example config is almost 1200 lines long, as the capabilities of Unbound are considerable, but we won’t need nearly as much. (If you are interested, any and all configurables can be found in the extensive manual page with :manpage:`unbound.conf`.
```

Comment:
s/As prerequisite/As a prerequisite/
s/from other devices than only/from devices other than/
s/the machine on which the instance is running/the machine on which Unbound is running/